tests/integration/DBase4Test.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+namespace org\majkel\dbase;
+
+use org\majkel\dbase\tests\utils\TestBase;
+
+/**
+ * Integration tests of dBase IV Format
+ *
+ * @author stokescomp
+ */
+class DBase4Test extends TestBase {
+
+    /**
+     * @medium
+     * @coversNothing
+     * @throws Exception
+     */
+    public function testReadDbase4() {
+        $dbf = Table::fromFile('tests/fixtures/dBase4.dbf');
```

Comment:
This test fails because it cannot find corresponding memo file (fbt)

Reply:
is that FPT format? How do you choose one over the other?
In readme I see:
##### Supported memo formats
 - DBT
 - FPT

In the failed test I see:
1) org\majkel\dbase\DBase4Test::testReadDbase4
Failed asserting that two strings are identical.
--- Expected
+++ Actual
@@ @@
-memo1
+▒▒
+memo1